modules/simple_forms_api/docs/rfcs/README.md

```diff
@@ -0,0 +1,105 @@
+# RFCs Directory
```

Comment:
Thanks for writing this README! Very helpful!

Reply:
My pleasure! I wasn't sure how familiar everyone was with them and anyone new isn't guaranteed to know either.